packages/snap/src/index.tsx

```diff
@@ -190,8 +207,9 @@ export const processRpcRequest: SnapRpcProcessor = async (
         importParams.encryptedZkCert,
         holder.encryptionPrivKey,
       );
+
       const schema = choseSchema(
-        decrypted.zkCertStandard as ZkCertStandard,
+        decrypted.zkCertStandard as KnownZkCertStandard,
```

Comment:
I think this is wrong. The snap should be able to handle future certificate types that are not known at compile time.